### PR TITLE
Add Documentation for offline option, This fixes #35

### LINKED ISSUE #35: Describe -offline in documentation
I just came across https://github.com/nextflow-io/nextflow/issues/665 by accident, which I didn't know about. I think this would be useful for a lot of our users (in Sweden on the Bianca cluster especially). Probably mostly by recommending that people export `NXF_OFFLINE=true` on such environments.

This can be added to the core cookiecutter documentation.

### PR DESCRIPTION
**Pull request by apeltzer**

### CODE REVIEW

**Review by ewels — COMMENTED** [review]
There's already a section for offline use under installation - see https://github.com/nf-core/cookiecutter/blob/master/%7B%7Bcookiecutter.pipeline_slug%7D%7D/docs/installation.md#offline-use

Might make more sense to put it there I think (I see this as a one-off task).

**Comment by apeltzer** [comment]
Adding this to the right file right now - please have a look :-)

File: {{cookiecutter.pipeline_slug}}/docs/installation.md [hunk 1]
```diff
@@ -41,3 +41,9 @@ unzip master.zip -d /my-pipelines/
 cd /my_data/
 nextflow run /my-pipelines/{{ cookiecutter.pipeline_slug }}-master
 ```
+
+If you are running the pipeline on an offline resource (e.g. a private cluster without internet connection available upon pipeline running), you may specify an environment variable as above to avoid download failure messages when starting the pipeline. Just add this to your environment or profile as above:
```
Comment (ewels) [inline {{cookiecutter.pipeline_slug}}/docs/installation.md]: This kind of repeats the previous text a little, then refers to "as above" when the example is below... 😉 

How about:

> To stop nextflow from looking for updates online, you can tell it to run in offline mode by specifying the following environment variable in your `~/.bashrc` file:

File: {{cookiecutter.pipeline_slug}}/docs/installation.md [hunk 2]
```diff
@@ -41,3 +41,9 @@ unzip master.zip -d /my-pipelines/
...
+
+```bash
+NXF_OFFLINE='TRUE'
```
Comment (ewels) [inline {{cookiecutter.pipeline_slug}}/docs/installation.md]: Should be `export NXF_OFFLINE='TRUE'` I think...

**Review by ewels — APPROVED** [review]

**Pull request merged**